scripts/release/prepare-release-from-ci.js

```diff
@@ -19,18 +18,21 @@ const run = async () => {
     const params = parseParams();
     params.cwd = join(__dirname, '..', '..');
 
+    const channel = params.releaseChannel;
+    if (channel !== 'experimental' && channel !== 'stable') {
+      console.error(
+        `Invalid release channel: "${channel}". Must be "stable" or "experimental".`
```

Comment:
May help clarify a tad to call out the param name too?
```suggestion
        `Invalid release channel (-r) "${channel}". Must be "stable" or "experimental".`
```

Reply:
Might be better still to follow precedent here and add this validation into `parse-params.js` e.g.
https://github.com/facebook/react/blob/a656ace8da6d005e49a750da3d9a3fd0fdcac76c/scripts/release/publish-commands/parse-params.js#L35-L46